python/packages/ai/teams/ai/planner/ai_history_options.py

```diff
@@ -0,0 +1,23 @@
+"""
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+"""
+
+from dataclasses import dataclass
+from enum import Enum
+
+
+class AssistantHistoryType(Enum):
```

Comment:
For this to be a string enum and work with intellisense it needs to also implement `str`, like `class AssistantHistoryType(str, Enum)`

Reply:
Seems the current code is just using it as enum. May inherit from `str` later if we have the requirement. I'm fine to inherit `str`, but personally prefer to make the implementation simpler.

By the way, intellisense works in my VS Code without inheriting `str`. I could take a look with you together offline if it doesn't work in your side.